.buildkite/pipeline.yml

```diff
@@ -98,6 +98,17 @@ steps:
           - build/*.xml
           - build/coverage*.out
 
+      - label: ":smartbear-testexecute: Run fips140=only unit tests"
+        key: unit-test-fips140-only
+        command: ".buildkite/scripts/fips140_test.sh"
```

Comment:
How is `fips140=only` passed into this? I am not seeing it in the diff.

Reply:
I made the env var declaration in buildkite explicit